### PR TITLE
fix: update institution name of g.educaand.es

### PR DESCRIPTION
**Pull request by Techie-Pi**
The ``g.educaand.es`` domain is used across all Andalusia starting 2021/2022 with the intention of unifying the corporate email addresses across the whole autonomous region (""state"").

This is referenced in the official page of Junta de Andalucía: https://www.juntadeandalucia.es/educacion/eaprendizaje/gsuite/

Cite:
> (...) The Consejería de Educación y Deporte de la Junta de Andalucía, after an agreement, has created the domain g.educaand.es for the management of these accounts for the Andalusian public schools. (...)

<details>
<summary>Original (Spanish)</summary>
La Consejería de Educación y Deporte de la Junta de Andalucía, tras el convenio firmado , ha creado el dominio g.educaand.es para la gestión de estas cuentas para los centros educativos andaluces de titularidad pública.
</details>

TLDR; This domain refered to a specific school instead of the whole autonomous region.

### CODE REVIEW

**Comment by Techie-Pi** [comment]
Cited text: [here](https://www.juntadeandalucia.es/educacion/eaprendizaje/gsuite/#:~:text=La%20Consejer%C3%ADa%20de%20Educaci%C3%B3n%20y%20Deporte%20de%20la%20Junta%20de%20Andaluc%C3%ADa%2C%20tras%20el%20convenio%20firmado%20%2C%20ha%20creado%20el%20dominio%20g.educaand.es%20para%20la%20gesti%C3%B3n%20de%20estas%20cuentas%20para%20los%20centros%20educativos%20andaluces%20de%20titularidad%20p%C3%BAblica.)
Agreement referenced in the issue and web: [here](https://www.juntadeandalucia.es/organismos/educacionydeporte/consejeria/transparencia/contratos-convenios/convenios/detalle/209930.html)

If further proof is needed in order to make this change, I could contact the required institutions

**Comment by philipto** [comment]
@Techie-Pi Thank you for the update. I changed the description to "Schools in Andalusia" to have more precise one. Pull request merged.
